hack/swagger-v2/main.go

```diff
@@ -25,6 +25,8 @@ import (
 	"k8s.io/kube-openapi/pkg/common"
 	"k8s.io/kube-openapi/pkg/validation/spec"
 
+	_ "k8s.io/code-generator"
+
```

Comment:
Could you move this to hack/swagger-v2/tools.go?
You can find an example here: https://github.com/kubeflow/katib/blob/master/hack/tools.go